package.json

```diff
@@ -13,12 +13,14 @@
   },
   "license": "AGPL-3.0",
   "dependencies": {
+    "angular-sanitize": "1.5.8",
```

Comment:
Good catch, it was an angular-markdown-editor dependency, but I do not use it. will be removed ...